sql/catalyst/src/main/java/org/apache/spark/sql/streaming/GroupStateTimeout.java

```diff
@@ -37,7 +37,9 @@
    * `map/flatMapGroupsWithState` by calling `GroupState.setTimeoutDuration()`. See documentation
    * on `GroupState` for more details.
    */
-  public static GroupStateTimeout ProcessingTimeTimeout() { return ProcessingTimeTimeout$.MODULE$; }
+  public static GroupStateTimeout processingTimeTimeout() {
```

Comment:
That's on purpose to provide more consistent APIs for Scala and Java. Let's just suppress them.